server/socket/socketManager.js

```diff
@@ -76,7 +76,17 @@ exports.socketManager = (io, socket) => {
     }
   });
 
-  socket.on('updatedRobotJobStatus', ({ jobId, status }) => {
-    socketHelperFunctions.updateRobotJobStatus(jobId, status);
+  socket.on('updatedRobotJob', ({ jobId, robotLogs }) => {
+    if (robotLogs.final_message === 'Execution completed') {
+      socketHelperFunctions.updateRobotJobStatus(
+        jobId,
+        robotLogs.robot_run.status === 'FAIL' ? 'failed' : 'successful'
```

Comment:
Why does there need to be a mapping here? Isn't it possible to set the status in the local client to failed instead of FAIL? That would make it probably easier to understand

Reply:
No, the status is set by the logging component from robotframework. We could do this mapping manually in the local client, but I'd prefer sending the data from the local client as raw as possible, so that we can work with it later on.